rizinorg/cutter
# Remove Breakpad Crash Reporting

**Your checklist for this pull request**


**Detailed description**

Crash dumps generated with breakpad were not made use of to an extent that would justify the extra maintenance overhead that it comes with. As Windows builds have recently been broken by it, now is a good time to retire it.

**Test plan (required)**

For all builds/platforms:
* Download an run the build and open a file, Cutter should work as normal
* Click File -> New Window
* Cutter should segfault and crash with the OS' default behavior

Tested:
- [x] macOS/x86_64
- [x] macOS/arm64
